test/runtests.jl

```diff
@@ -1,4 +1,5 @@
-using PowerModelsProtection
+# using PowerModelsProtection
+include("../src/PowerModelsProtection.jl")
```

Comment:
Why is using broken / not being used? In runtests you definitely want to use `using` to confirm the package is importing correctly.

Reply:
@pseudocubic Sorry I removed that.